scripts/plugins/storage/sqlite/init.sql

```diff
@@ -617,6 +617,13 @@ INSERT INTO foglamp.configuration ( key, description, value )
               ' { "plugin" : { "type" : "string", "value" : "omf", "default" : "omf", "description" : "Module that OMF North Plugin will load" } } '
             );
 
+-- dht11 - South plugin for DHT11 - C
```

Comment:
We don't need to add a new entry in the Database.

The caller of this plugin will get the the plugin default config and then call the configuration manager creating/updating the configuration.

Anyway, the DB addition has to be added to postgres as well.

And we need upgrade/downgrade files for both.

I suggest checking the sending_process configuration process.

Soon we will remove unnecessary DB entries.

As long as this is a new "component" we can start with no DB configuration

Reply:
IIRC, if I didn't provide configuration in init.sql, there was some failure during dht11 south service initialization. "add_service" Rest API is still to be implemented for C++ services in FOGL-1568. Once that is done, we won't need these changes in init.sql files. And I will need to understand upgrade/downgrade files/scripts.

Reply:
For the "configuration" you can do this:

managemnet_client:

addCategory(category, true))

example in:

develop/C/tasks/north/sending_process/sending.cpp#L522

This way you don't need to put it into foglamp.configuration

Reply:
Will remove entries from init.sql in one go once we have the fix for FOGL-1568, the way we already have done for python plugins. Also the plugins shall be added by wizard (as per FOGL-1522), so I believe the add service and plugin discovery all shall be done by that itself.

Reply:
ok, right now we have to keep the plugin initial  plugin configuration int the DB